static/js/message_list_view.js

```diff
@@ -552,6 +552,7 @@ MessageListView.prototype = {
                 // mention text to show the user's current name,
                 // assuming that you're not searching for text
                 // inside the highlight.
+                user_id = parseInt(user_id, 10);
```

Comment:
Perhaps this conversion should be moved into `get_user_id_for_mention_button` (which would then return `"*" | number`).

Reply:
yeah, that's fair, since in the `email` path you already have an int...changed